ts/store/reducers/loading.ts

```diff
@@ -16,7 +15,8 @@ export type LoadingState = Readonly<
 export const INITIAL_STATE: LoadingState = {
   PIN_CREATE: false,
   PROFILE_LOAD: false,
-  PROFILE_UPDATE: false
+  PROFILE_UPDATE: false,
+  MESSAGES_LOAD: false
```

Comment:
I was wondering why, instead of having a `LoadingState` and an `ErrorState`, we don't just have a `State`?

```ts

type ActionState = { kind: "empty" } | { kind: "loading" } | { kind: "error", error: string }; 

interface State {
  [key]: ActionState
}
```